pyproject.toml

```diff
@@ -31,6 +31,7 @@ starlette-oauth2-api = "^0.2.6"
 httpx = "^0.20.0"
 
 # fasttext = "^0.9.2"
+regex = "2022.3.2"
```

Comment:
I was keep getting errors on pytest and it seemed like a similar issue to [this](https://github.com/domainaware/parsedmarc/issues/298) so I downgraded the version of regex.